### ISSUE TITLE
Failled to add user and code

### ISSUE THREAD

**Issue by sorinpetrus3** [nielsfaber/alarmo#424]
### Alarmo Version

v1.8.6

### HA Version

2022.3.5

### Bug description

When i want to add a user and password i get this error 
" Something went wrong!
Message format incorrect: extra keys not allowed @ data['is_admin']

Response error: 400

Please [report](https://github.com/nielsfaber/alarmo/issues) the bug "

### Steps to reproduce

just i install the alarmo addon and i test, everything it's working fine i can arm and disarm the alarm without code but i can't add code

**Comment by nielsfaber** [comment]
`is_admin` has been removed from the code in november.
Your alarmo instance is out of date.
Check the version number in the top right corner.

**Comment by sorinpetrus3** [comment]
You are right I have version 1.7.2 but how I can update this? I remove from integration and after from hacs. I restart ha and downloaded again in hacs version 1.8.6 , restart ha and add in integration and is still version 1.7.2. What can I do??

**Comment by nielsfaber** [comment]
If the Alarmo configuration panel version is out of date, you need to clear your browser cache.

**Comment by sorinpetrus3** [comment]
Yes, you are right again, I clean the cache and everything it's working fine now. Thank you and sorry for this, can be closed now. Thank you!
